db.go

```diff
@@ -28,13 +28,19 @@ const (
 	//   - may be faster is some use-cases (random reads - indexer)
 	//   - use boltdb build tag (go build -tags boltdb)
 	BoltDBBackend BackendType = "boltdb"
-	// RocksDBBackend represents rocksdb (uses github.com/cosmos/gorocksdb)
+	// RocksDBBackend represents rocksdb (uses github.com/linxGnu/grocksdb)
 	//   - EXPERIMENTAL
 	//   - requires gcc
 	//   - use rocksdb build tag (go build -tags rocksdb)
 	RocksDBBackend BackendType = "rocksdb"
-
+	// BadgerDBBackend represents badgerdb (uses github.com/dgraph-io/badger)
+	//   - EXPERIMENTAL
+	//   - use badgerdb build tag (go build -tags badgerdb)
 	BadgerDBBackend BackendType = "badgerdb"
+	// PebbleDBDBBackend represents pebble (uses github.com/cockroachdb/pebble)
+	//   - EXPERIMENTAL
+	//   - use pebble build tag (go build -tags pebbledb)
```

Comment:
I believe this is pure go so no need for a build flag.

Reply:
We tag all of them currently (badger, bolt, and pebble)

Reply:
I think that if we don't tag, then we end up making chain binaries that use more than one db.  This could (or might not) be desirable....

Reply:
if we want to make it default, then no flag.  But I think default is like maybe next month.

Cause it is 10-27x less intense on the disk, no joke.

Reply:
https://twitter.com/gadikian/status/1553351762894917632?s=20&t=eXmh9UbBSYDnBO3kRFbh2w

Reply:
27x aint a joke, true

Reply:
eh, need to do more benchmarking.  its.... wierd. 

But:

* near certain drives will last longer
* near certain will be faster with cw
* near certain will enable running on much cheaper nodes

Reply:
I took the tags off.  Not sure what'll happen with tests and such